modules/installation-configuration-parameters.adoc

```diff
@@ -1531,6 +1531,46 @@ within link:https://azure.microsoft.com/en-us/global-infrastructure/regions[a re
 |Enables the encryption of the virtual machine guest state for compute nodes. This parameter can only be used if you use Confidential VMs.
 |`VMGuestStateOnly` is the only supported value.
 
+|controlPlane:
+  platform:
+    azure:
+      identity:
+        type:
+|The type of identity used for control plane virtual machine.
+The `UserAssigned` identity is a standalone Azure resource provided by the user and assigned to control plane virtual machines.
+Identity can only be set for control plane nodes.
```

Comment:
We can remove this line. It is inaccurate (out of date). I have updated openshift/installer#9717 to remove it from the installer code.